elements/pfe-tabs/src/pfe-tab-panel.scss

```diff
@@ -1,11 +1,17 @@
 @import "../../pfe-sass/pfe-sass";
+@import "../../../docs/themes/kube/assets/src/sass/_vars.scss";
```

Comment:
This is being deprecated (imminently) so we can't pull assets from the hugo generator.